rust/main/agents/relayer/src/main.rs

```diff
@@ -18,6 +18,7 @@ mod memory_profiler;
 
 #[tokio::main(flavor = "multi_thread", worker_threads = 20)]
 async fn main() -> Result<()> {
+    println!("Relayer starting up...");
```

Comment:
is this before tracing is available? same goes for all the println usage throughout this PR

Reply:
It is correct. Tracing is not initialised at this point. So, using `println!`. I added a comment.